templates/CRM/common/paymentBlock.tpl

```diff
@@ -104,12 +104,19 @@
       {else}
         {capture assign='profilePathVar'}{/capture}
       {/if}
+      {if $contribID}
```

Comment:
I don't think we need any of the changes on this form if we are not doing the edit overload thing